.gitignore

```diff
@@ -29,9 +29,14 @@ build/Release
 # Dependency directories
 node_modules
 jspm_packages
+yarn.lock
```

Comment:
I don't think there's actually anything wrong with having the `yarn.lock` file on GitHub